.gitlab/common/test_infra_version.yml

```diff
@@ -1,4 +1,9 @@
+# File generated by inv buildimages.update-test-infra-definitions
+# Please do not edit this file manually
+# To update the test-infra-definitions version, run `inv buildimages.update-test-infra-definitions --commit-sha <commit_sha>` [--is-dev-image]
+
+
 ---
 variables:
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 221bbc806266
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: ''
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES: ed2e00040283
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: -dev
```

Comment:
To be updated with the version from main branch of test-infra before merging

Reply:
This is what I had in mind. Indeed, I have to merge first https://github.com/DataDog/test-infra-definitions/pull/1316 and then this PR.